src/NuGet.Server/Default.aspx

```diff
@@ -1,49 +1,247 @@
+        }
+        .nav-sidebar > li > a {
+          padding-right: 20px;
+          padding-left: 20px;
+        }
+        .nav-sidebar > .active > a,
+        .nav-sidebar > .active > a:hover,
+        .nav-sidebar > .active > a:focus {
+          color: #fff;
+          background-color: #428bca;
+        }
+        .main {
+          padding: 20px;
+        }
+        @media (min-width: 768px) {
+          .main {
+            padding-right: 40px;
+            padding-left: 40px;
+          }
+        }
+        .main .page-header {
+          margin-top: 0;
+        }
+        .placeholders {
+          margin-bottom: 30px;
+          text-align: center;
+        }
+        .placeholders h4 {
+          margin-bottom: 0;
+        }
+        .placeholder {
+          margin-bottom: 20px;
+        }
+        .placeholder img {
+            display: inline-block;
+            border-radius: 50%;
+        }
+        div.panel tr.prerelease {
+            font-style: italic;
+        }
+        div.panel tr.latest {
+            font-weight: bold;
+        }
+    </style>
+</head>
+<body>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="/">Nuget Server</a>
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+          <ul class="nav navbar-nav">
+            <li><a href="<%= VirtualPathUtility.ToAbsolute("~/nuget/Packages") %>">Feed</a></li>
+            <% if (Request.IsLocal) { %>
+              <li><a href="<%= VirtualPathUtility.ToAbsolute("~/nugetserver/api/clear-cache") %>">Clear Cache</a></li>
+            <% } %>
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+     <div class="container">
+      <div class="row">
+        <div class="col-sm-3 col-md-2 sidebar">
+          <ul class="nav nav-sidebar">
+            <li class="active"><a href="#">Overview <span class="sr-only">(current)</span></a></li>
+          </ul>
+          <ul class="nav nav-sidebar">
+              <%foreach (var package in packages)
+                {
+                    var first = package.First();%>
+                 <li><a href="#<%= package.Key.Id.Replace(' ', '_')%>"><%= first.Title %></a></li>     
+              <%}%>
+          </ul>
+        </div>
+        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+            <h1>ahd Nuget Server</h1>
+            <div>
+                <h2>You are running NuGet.Server v<%= typeof(NuGet.Server.DataServices.ODataPackage).Assembly.GetName().Version %></h2>

```

Comment:
Error: The type or namespace name 'ODataPackage' does not exist in the namespace 'NuGet.Server.DataServices' (are you missing an assembly reference?)